api/src/main/java/com/nicknathanjustin/streamercontracts/users/UsersApiController.java

```diff
@@ -33,4 +36,19 @@ public ResponseEntity user(@Nullable final OAuth2Authentication auth) {
 
         return new ResponseEntity(HttpStatus.FORBIDDEN);
     }
+
+    @RequestMapping(path = "/{twitchUsername}", method = RequestMethod.GET)
+    @JsonView(View.PublicUser.class)
+    public ResponseEntity<UserDto> publicUser(@PathVariable("twitchUsername") @NonNull final String twitchUsername) {
+        final TwitchUser twitchUser = userService.getTwitchUserFromUsername(twitchUsername);
+        final Optional<UserModel> optionalUserModel = userService.getUser(twitchUsername);
+        final UserModel userModel = optionalUserModel.orElse(null);
+
+        if (twitchUser != null && userModel != null) {
+            final UserDto userDto = new UserDto(twitchUser, userModel);
```

Comment:
Again, doesn't need to be done now, but I'm still not convinced that a UserDto and a TwitchUser obj is the right way to go. We have duplicate fields in both classes. There's definitely a better way to do this.